smtp/client.ts

```diff
@@ -1,87 +1,45 @@
-const { SMTP, state } = require('./smtp');
-const { Message, create } = require('./message');
-const addressparser = require('addressparser');
+// @ts-ignore
```

Comment:
why `ts-ignore` ?

Reply:
there's no `@types` module for `addressparser` and i didn't feel like writing the headers :P

Reply:
you could write a declaration file for address parser that exports an `any` type. this makes it more explicit to others what you are specifically ignoring and you don't need a `ts-ignore` for the entire file

Reply:
i could, but then i'd have to modify the tsconfig, and UGH MOM FINE i'll clean my bedroom!!! :)

Reply:
ha.

Reply:
done :)